docs/mkdocs.yml

```diff
@@ -39,13 +39,17 @@ extra_css:
 # Navigation bar
 nav:
   - Home: index.md
+  - 3-minute Guide: 3_minutes.md
   - User Guide: user_guide.md
   - Tutorials: tutorials/
   - API Reference: reference/ # generated by gen-files + literate-nav
 
 # Plugins
 plugins:
   - search
+  - exclude-search:
+      # Exclude all files not listed in the nav section nor referenced by other files.
+      exclude_unreferenced: true
```

Comment:
This fixes the removal of the SUMMARY file + a bunch of other annoying files and makes search much much cleaner 😄